.github/workflows/maven-central-release.yml

```diff
@@ -0,0 +1,39 @@
+name: Maven Central Relase
+on:
+  workflow_dispatch:
+    inputs:
+      releaseVersion:
+        description: 'Release version'
```

Comment:
Next version can optionally be provided in the UI. If not - the one from pom.xml will be used (-SNAPSHOT is dropped automatically)
![image](https://user-images.githubusercontent.com/2108093/127923988-b3d402b1-4612-4b6b-9a77-b8f81cb4c385.png)